pipelines/pipelines/kafka_utils.py

```diff
@@ -64,9 +64,15 @@ def create_topics() -> None:
             admin_client = admin.KafkaAdminClient(
                 bootstrap_servers=config.KAFKA_BOOTSTRAP_SERVER,
             )
-        except Exception:
-            logger.exception("Failed to create KafkaAdminClient.")
-            raise
+        except Exception:  # noqa
+            try:
+                # idk why, locally it works only since second initialization
+                admin_client = admin.KafkaAdminClient(
+                    bootstrap_servers=config.KAFKA_BOOTSTRAP_SERVER,
+                )
+            except Exception:
+                logger.exception("Failed to create KafkaAdminClient.")
+                raise
```

Comment:
Maybe KafkaAdminClient has retry function? https://kafka-python.readthedocs.io/en/master/apidoc/KafkaAdminClient.html
Or simply add for loop.